sysbuild/CMakeLists.txt

```diff
@@ -333,11 +333,12 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
           set_config_bool(mcuboot CONFIG_NORDIC_QSPI_NOR_XIP y)
         endif()
       else()
-        set(${DEFAULT_IMAGE}_SIGNING_SCRIPT "${ZEPHYR_NRF_MODULE_DIR}/cmake/sysbuild/image_signing.cmake" CACHE INTERNAL "MCUboot signing script" FORCE)
+        UpdateableImage_Get(images "ALL")
+        foreach(image ${images})
+          set(${image}_SIGNING_SCRIPT "${ZEPHYR_NRF_MODULE_DIR}/cmake/sysbuild/image_signing.cmake" CACHE INTERNAL "MCUboot signing script" FORCE)
+        endforeach()
```

Comment:
should probably use proper names for these because `UpdateableImage_Add` might be used more in future, for multiple bootloaders